packages/grafana-ui/src/components/uPlot/config/UPlotConfigBuilder.ts

```diff
@@ -232,10 +232,16 @@ export class UPlotConfigBuilder {
       if (this.bands?.length) {
         config.bands = this.bands;
         const killFill = new Set<number>();
+
         for (const b of config.bands) {
           killFill.add(b.series[1]);
         }
 
+        // don't remove fills of series that have their own fillBelowTo defined (pseudo-stacking case)
+        for (const b of config.bands) {
+          killFill.delete(b.series[0]);
+        }
```

Comment:
Is there a way to add a unit test for this condition / scenario?

Reply:
i wanted to see first if we wanted to instead remove all the magic and just leave it up to users to explicitly disable fills when they set fillBelowTo. in that case this would just go away.